Gemfile

```diff
@@ -155,8 +155,8 @@ group :development do
 
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem "spring"
+  gem "spring-commands-rspec"
```

Comment:
Could you add a note to the readme on how to use it?

Is it compatible with running rspec in parallel? If not, which do we want to pick?

Reply:
ran parallel test and it all works fine!